src/NUnitFramework/tests/Constraints/CollectionContainsConstraintTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanTestContentsOfObjectList()
             Assert.That(list, new CollectionContainsConstraint(item));
         }
 
-#if !SILVERLIGHT
+#if !SILVERLIGHT && !PORTABLE
```

Comment:
A couple of the test projects were targeting Silverlight. I switched them to targeting Portable so that they could be tested under different environments.